apps/coscrad-frontend/src/store/slices/resources/playlists/thunks/fetchPlaylists.ts

```diff
@@ -0,0 +1,10 @@
+import { buildResourceFetchActionPrefix } from '../../../utils/build-resource-fetch-action-prefix';
```

Comment:
Delete this duplicate. You already have a `thunks.ts`. You don't need a `thunks` directory or a second `fetchPlaylists` file.